src/controllers/requestController.js

```diff
@@ -0,0 +1,22 @@
+import RequestServices from '../services/requestService';
+import Response from '../utils/response';
+
+/**
+ * Class for managing trip request
+ */
+export default class RequestController {
+  /**
+   * @param {req} req - request obiect
+   * @param {res} res - res object object
+   * @returns {object} - return object
+   */
+  static async multiCityRequest(req, res) {
+    try {
+      const data = await RequestServices.multiCityRequest(req);
+
+      Response.success(res, data);
```

Comment:
you can use object destructuring here e.g `const { success, badRequest} = Response`
so you can have `success(res, data, 201)`. You can do this in other places where this applies. I think this will make your code cleaner

Reply:
Ok I will do that